lib/hammer_cli_katello/content_view_purge.rb

```diff
@@ -48,18 +48,19 @@ def option_sources
       sources
     end
 
+    # rubocop:disable Layout/LineLength
```

Comment:
If you wouldn't mind, please remove this linelength limit and split the long lines 54 and 63 into smaller lines.